.gitignore

```diff
@@ -1,8 +1,10 @@
 /.bundle/
 /doc/
 /log/*.log
+/node_modules/
 /pkg/
 /tmp/
 /test/dummy/log/*.log
 /test/dummy/tmp/
+/yarn.lock
```

Comment:
This project doesn't have a `package.json` or other Node or Yarn integration.

What are the motivating scenarios behind this change?